modules/new_user/IDs.csv

```diff
@@ -0,0 +1,11 @@
+WELCOME_CHANNEL_ID=785232784225861643
```

Comment:
CSV stands for comma separated values. If you're not using CSV format in your CSV files, you might as well use a .txt file.